src/app/plugins/platform/Docker/spawnComposer.ts

```diff
@@ -53,15 +55,7 @@ async function spawnComposer(
 
   return execa(
     'docker',
-    [
```

Comment:
Unless you've got a strong preference I lean a bit more toward keeping this broken across multiple lines to keep the diffs simpler whenever this is tweaked.

Reply:
This is the format mandated by our ESLint configuration since we run prettier on all files.